invenio_rdm_records/records/api.py

```diff
@@ -117,7 +118,8 @@ class CommonFieldsMixin:
             attrs=['id', 'title'],
             pid_field=Vocabulary.pid.with_type_ctx('resourcetypes'),
             cache_key='resource_type',
-            relation_field='resource_type'
+            relation_field='resource_type',
+            value_check=dict(props=dict(relation_type='linkable', subtype="publication-section")),
```

Comment:
Some module somewhere needs to bump their dependencies so that invenio-rdm-records has the invenio-records version that supports this...

Reply:
Yes, if we release a minor version we will have to bump it.